http-server/src/main/java/org/triplea/server/http/ServerApplication.java

```diff
@@ -80,18 +81,17 @@ public void run(final AppConfig configuration, final Environment environment) {
         .forEach(controller -> environment.jersey().register(controller));
   }
 
-
   private List<Object> exceptionMappers() {
     return ImmutableList.of(
         new IllegalArgumentMapper(),
         new IncorrectApiKeyMapper(),
-        new ApiKeyVerificationLockOutMapper());
+        new ApiKeyLockOutMapper());
```

Comment:
Rename here as lockout is not just on verification but can also be during registration